.github/pull_request_template.md

```diff
@@ -6,17 +6,17 @@ right people know. There are language-specific teams for reviewing recipes.
 
 | Language        | Name of review team           |
 | --------------- | ----------------------------- |
-| python          | `@conda-forge/help-python`    |
-| python/c hybrid | `@conda-forge/help-python-c`  |
-| r               | `@conda-forge/help-r`         |
+| c/c++           | `@conda-forge/help-c-cpp`     |
```

Comment:
I did note there are a few more [help teams](https://github.com/orgs/conda-forge/teams?query=help-) (specifically arch related)... should these be added? It seems like the cdt team isn't worth adding :P